core/textile/hub/hub_auth.go

```diff
@@ -67,16 +68,18 @@ type HubAuth interface {
 }
 
 type hub struct {
-	st  store.Store
-	kc  keychain.Keychain
-	cfg config.Config
+	st               store.Store
+	kc               keychain.Keychain
+	cfg              config.Config
+	fetchTokensMutex *sync.Mutex
 }
 
 func New(st store.Store, kc keychain.Keychain, cfg config.Config) *hub {
 	return &hub{
-		st:  st,
-		kc:  kc,
-		cfg: cfg,
+		st:               st,
+		kc:               kc,
+		cfg:              cfg,
+		fetchTokensMutex: &sync.Mutex{},
```

Comment:
This is already in alpha branch right? Looks harmless but flagging in case some merge issue is there.

Reply:
Added this additional lock apart from the one in master given that the hub is accessed directly from the space service when doing `GetSessionTokens`, which sometimes triggered both healthcheck and that to go off at the same time